src/Cli/dotnet/CommonOptions.cs

```diff
@@ -91,6 +95,18 @@ public static Argument DefaultToCurrentDirectory(this Argument arg)
                 "--interactive",
                 CommonLocalizableStrings.CommandInteractiveOptionDescription);
 
+        public static Option ArchitectureOption(bool includeShortVersion = true) =>
+            new ForwardedOption<string>(
+                includeShortVersion ? new string[] { "--arch", "-a" } : new string[] { "--arch" },
+                CommonLocalizableStrings.ArchitectureOptionDescription)
```

Comment:
How about a proper long and short form for the Architecture option?

```suggestion
            new ForwardedOption<string>(
                includeShortVersion ? new string[] { "--architecture", "-a" } : new string[] { "--architecture" },
                CommonLocalizableStrings.ArchitectureOptionDescription)
```

##### NOTES

- If there's a way to add `--arch` as an alias of the long form `--architecture`, that would make this more friendly and intuitive!!!
- See examples below…

Reply:
We've decided to stick with the short versions per the spec.